vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -1429,6 +1616,538 @@ private void fetchWorkflows(HttpServerExchange exchange) {
     return Stream.empty();
   }
 
+  private void load(HttpServerExchange exchange, UnloadedData unloadedData) {
```

Comment:
I've tried to break it up, but the early returns make it hard. If you see other splits that work, let me know.

Reply:
It seems like it would be easy to extract the queries into separate methods? Again, that would be very helpful for testing too, though having separate DAO classes would be even better... 😁

Reply:
DAOs are singletons and I'm not doing that.

Here they are separated, but I think it makes it harder to understand what's going on.

Reply:
It's a lot easier for me to follow now as a reviewer